docs/usage.adoc

```diff
@@ -114,9 +116,171 @@ Similar to disks, volumes can also be private or shared among nodes. Private vol
 
 A `size` and `class` params can be specified for every volume. If you don't specify them, the default values will be used. To specify default values use the <<installation#values,`values.yaml`>> file during the installation stage.
 
-==== sidecars & initContainers
-https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Init containers]
-  and sidecars can be added to the cluster.
+==== initContainers
+
+In the context of Xp7Deployment, `initContainers` are containers launched before the XP container itself, as part of the deployment lifecycle. This mechanism is particularly useful to set up the environment or parameters for the main application or to ensure all prerequisites are in place before the actual application starts.
```

Comment:
This is as abstract as k8s documentation can be.
For XP it is mainly used to add applications to deploy directory. I think it should be mentioned here